mu-plugins/blocks/global-header-footer/postcss/header/header.pcss

```diff
@@ -90,6 +90,38 @@ html {
 	}
 }
 
+.global-header__alert-banner {
+	--wp-global-header--link-color: #3e58e1;
+
+	background-color: #eff2ff;
+	color: #1e1e1e;
```

Comment:
There's a var for this in the News and Parent themes (albeit with different names), but nothing available universally. The other colors are similar.

I haven't seen a canonical location where all colors get defined for custom blocks, block themes, the global header/footer, and classic themes. LMK if I missed it, though.

Reply:
Right, I don't think we've agreed on a strategy for that. I assume that the parent theme is supposed to be the eventual source of truth.

What about having a fallback for now: `var(--wp--preset--color--charcoal-1, #1e1e1e)`?

Reply:
Doh, it looks like that already exists in `_common.pcss` and I just missed it. fixed in 7198958. thanks!

Reply:
> I assume that the parent theme is supposed to be the eventual source of truth.

That's correct, and those colors are documented here: https://github.com/WordPress/wporg-parent-2021/#colors. Since the global header/footer need to be rendered on old sites too, the base colors we need are redefined in `_common.pcss`. So you could also move over `--wp--preset--color--blueberry-4` for the `#eff2ff` value.

Reply:
👍🏻  fixed in ac660e528fbc409c2b2b9c3abc859aeeba68c89e